crates/dbsp/src/operator/mod.rs

```diff
@@ -27,7 +27,7 @@ mod integrate;
 mod join;
 pub mod join_range;
 mod neg;
-mod neighborhood;
+pub mod neighborhood;
```

Comment:
I think this will break `cargo doc` (which, btw, must be added to CI)

Reply:
Hmm, it fixed `cargo doc` for me: without this change, the references to `Neighborhood` and `NeighborhoodDescr` in documentation comments caused `cargo doc` errors because those are private types.

Reply:
I think I exported them explicitly to fix that. I'll double check.

Reply:
> I think I exported them explicitly to fix that. I'll double check.

Yeah, I saw that they were each exported explicitly, but `cargo doc` still gave me the error and didn't generate any documentation for those types. I admit that this seems weird.